docs/how-to/build-interacting-with-the-network.md

```diff
@@ -5,20 +5,29 @@ description: Learn how to interface with the Filecoin network.
 
 # Interacting with the Network
 
-**TODO: Replace content.** - Note that second-level headers are surfaced in the left nav. 
+This page outlines various options for connecting to networks while building and operating your service or application.
+
+Each Filecoin-based service or application will need to use at least one Filecoin node that maintains consensus. All interactions with the network must flow through an up-to-date node: sending and receiving market deals, sending and receiving data, and more.
+
+You can run this node yourself or choose a hosted option. Running your own node offers more decentralized control, but it requires non-trivial time and resources. Choosing a hosted node makes it easier to get started quickly, but you’ll need to trust the hosting provider.
 
 ## Local networks
 
-Some content!
+Here are some ways to spin up a network, for quick testing and development:
+- [Filecoin Node Hosting Quick Start](https://paper.dropbox.com/doc/Filecoin-Node-Hosting-Quick-Start-7RBPjls1Bz6WLkT634NEN) for k8s and Docker
+- [Containerized devnet using mocked sectorbuilder](https://github.com/textileio/lotus-devnet) (aka mocked mining)
+- [Additional containers & virtualization scripts](https://github.com/filecoin-project/docs/wiki#containers--virtualization)
 
 ## Node APIs & API endpoints
 
-Some more content!
+Coming soon.
+
+## Running your own remote nodes or networks
 
-## Running your own nodes
+The same containerization scripts for [local networks](#local-networks) can also be used to run your own remote nodes or networks.
 
-Some more more content!
+TODO: Additional tips.
 
 ## Testnets / Mainnet
```

Comment:
Just confirming you intended to leave this as a level 2 header and have it surfaced in the nav?

Reply:
Yes for this situation — again because it links to specific references that a dev might look up multiple times while designing/setting up an application.